src/kindlemint/agents/automation_coordinator.py

```
@@ -0,0 +1,682 @@
+    def _calculate_execution_time(
+        self, start_time: str, end_time: Optional[str]
+    ) -> float:
+        """Calculate workflow execution time in seconds"""
+        if not end_time:
+            return 0.0
+
+        try:
+            start = datetime.fromisoformat(start_time)
+            end = datetime.fromisoformat(end_time)
+            return (end - start).total_seconds()
+        except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)